packages/ui-kit/src/components/Log/useLog.ts

```diff
@@ -12,30 +12,32 @@ export const useLog = (): Log => {
 
   // Build the log methods with only supported console methods based on the log level.
   // Supported log methods are: "error | warn | info | debug"
-  const error = function (...args: any[]) {
+  const error = useCallback(function (...args: unknown[]) {
```

Comment:
In order to pass `log` as a dependency for useEffects it is necessary to memoize the functions as otherwise they change in every re-render causing triggering useEffects in every re-render